xdotool/functions

```diff
@@ -55,14 +55,25 @@ function zsh-notify() {
 
     zstyle -s ':notify:' "$type"-sound notification_sound
     zstyle -s ':notify:' "$type"-icon icon
+    zstyle -s ':notify:' app-name app_name
+    zstyle -s ':notify:' expire-time expire_time
 
     title=$(notification-title "$type" time_elapsed "$time_elapsed")
 
     if [[ -n "$icon" ]]; then
         icon_option="-i $icon"
     fi
 
-    notify-send ${=icon_option} "$title" "$message"
+    if [[ -n "$app_name" ]]; then
```

Comment:
I think we can go all-in and just make it so there's always an `-a` option for `notify-send`, either the first word of the command or a pre-set one.

```suggestion
    if [[ -z "$app_name" ]]; then
        app_name_option="-a ${message%% *}"
        message="${message#* }"
    else
        app_name_option="-a $app_name"
    fi
```